llmfoundry/data/finetuning/tasks.py

```diff
@@ -243,17 +252,21 @@ def slice_out_last_turn(
         messages_through_current_turn: list[dict[str, str]],
         conversation_through_previous_turn: str,
     ) -> tuple[str, str]:
+        chat_template = None if tokenizer.chat_template is not None else _DEFAULT_CHAT_TEMPLATE
+
```

Comment:
nit: can you log that we're using the default chat template

Reply:
I don't want to add a log here because it will run for every row in the dataset :)